demo/src/components/Header.astro

```diff
@@ -2,14 +2,17 @@
 import { getLocalePath, t } from "i18n:astro";
 import LocaleSwitcher from "./LocaleSwitcher.astro";
 ---
-    <header
-      class="bg-gray-100 p-2 border-b border-gray-300 flex items-center justify-between"
-    >
-      <div class="font-bold text-xl">astro-i18n demo</div>
-      <div class="flex items-center justify-between space-x-6">
-        <a href={getLocalePath("/")}>{t("home")}</a>
-        <a href={getLocalePath("/about")}>{t("about")}</a>
-        <a href={getLocalePath("/blog")}>{t("blog")}</a>
-      </div>
-      <LocaleSwitcher />
-    </header>
\ No newline at end of file
+<header class="bg-gray-300 p-2 sticky top-0 flex items-center justify-between flex-col sm:flex-row">
+  <a href="https://github.com/astrolicious/i18n/tree/main/demo">
+    <div class="flex items-center p-3">
+      <img src="../../src/content/astrolicious.png" alt="astrolicious" class="rounded-lg mr-3" >
```

Comment:
You can't use like so. Move the image to the public folder and use an absolute path